slip-0044.md

```diff
@@ -1265,6 +1265,7 @@ Coin type | Path component (`coin_type'`) | Symbol | Coin
 99999998 | 0x85f5e0fe | FLUID  | [Fluid Chains](https://www.fluidchains.com)
 99999999 | 0x85f5e0ff | QKC    | [QuarkChain](https://www.quarkchain.io)
 608589380 | 0xa4465644 | FVDC | [ForumCoin](https://fvd.nl)
+1179993420 | 0x4655454c |        | [Fuel](https://www.fuel.network)
```

Comment:
Hex value should be `0xc655454c`

Reply:
Ah right haha I copy-pasted the value without the offset by accident, good catch!

Reply:
Fixed in 741d8fe

Reply:
thank you